src/library/librarycontrol.cpp

```diff
@@ -166,6 +166,29 @@ LibraryControl::LibraryControl(Library* pLibrary)
                 &LibraryControl::slotMoveFocus);
     }
 
+    // Controls to move tracks on playlist
+    // Track move controls (emulate Alt+up/down button press)
+    m_pMoveTrackForward = std::make_unique<ControlPushButton>(ConfigKey("[Library]", "MoveTrackForward"));
+    m_pMoveTrackBackward = std::make_unique<ControlPushButton>(ConfigKey("[Library]", "MoveTrackBackward"));
+    m_pMoveTrack = std::make_unique<ControlEncoder>(ConfigKey("[Library]", "MoveTrack"), false);
```

Comment:
I suggest we make it
MoveTracks | ~Up | ~Down

just FYI: we aim to use snake_case for new controls.
Here PascalCase is okay as it it's consistent with the closely related controls.

Reply:
done

Reply:
> MoveTracks | ~Up | ~Down

Ha, sorry that was not clear, I meant:
MoveTrack**s**
MoveTrack**s**Up
MoveTrack**s**Down